Simplenote/src/main/java/com/automattic/simplenote/NoteEditorFragment.java

```diff
@@ -844,7 +844,9 @@ protected void showMarkdown() {
             new Runnable() {
                 @Override
                 public void run() {
-                    requireActivity().invalidateOptionsMenu();
+                    if(getActivity() != null) {
+                        getActivity().invalidateOptionsMenu();
+                    }
```

Comment:
```suggestion
                    if (!isDetached()) {
                        requireActivity().invalidateOptionsMenu();
                    }
```

If we're trying to avoid calling `invalidateOptionsMenu()` when the fragment is detached, would using `isDetached()` be better?